data/json/items/ammo.json

```diff
@@ -524,7 +524,7 @@
     "id": "120mm_HEAT",
     "price": 999999,
     "price_postapoc": 1000,
-    "name": { "str": "120mm HEAT" },
+    "name": { "str_sp": "120mm HEAT" },
```

Comment:
```suggestion
    "name": { "str": "120mm HEAT round" },
```

"HEAT rounds" is a good pluralization.

Reply:
True. I'm not sure if it's worth it lengthening it in the "what does this gun use" context, though. Something to be discussed.

Reply:
I am inclined to not add the word.